Simulation Validation/Validation of test statistics simulation.md

```diff
@@ -0,0 +1,202 @@
+---
+jupytext:
+  cell_metadata_filter: -all
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.13.7
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+  
+---
+This notebook validates the p-value generation functions `generate_p_value` while the alternative hypothesis is true by comparing:
+* sampling raw observations from the metric's distribution, calculating its test statistics and p-value, with
+* directly sampling test statistics and calculating p-values from its distribution 
+
+Parameters
+* δ: MDE
+* σ: standard deviation
+* n: sample size
+* size: number of replications
+
+## Numeric Metrics
+```python
+import numpy as np
+from scipy.stats import nct, ks_2samp
+from itertools import product
+from tqdm import tqdm
+import pandas as pd
+import seaborn as sns
+
+
+def obs_sample(δ, σ, n, size):
+    """
+    This function samples from normally distributed numeric metric's observations under true alternative hypothesis and calculates the samples' p-values
+
+    Attributes:
+    δ: the mean's shift from null hypothesis
+    σ: standard deviation
```

Comment:
```suggestion
    δ: delta, the mean's shift from null hypothesis
    σ: sigma, standard deviation
```
?

Reply:
is there a way to alias on the plots though? i am okay if we decide not to do this